build.rs

```diff
@@ -1,32 +1,18 @@
 use std::{env, fs, io, process};
 
-use build_util::link_visitor::{
-    syn::{self, visit_mut::VisitMut},
-    Link,
-};
 use camino::{Utf8Path, Utf8PathBuf};
 use quote::ToTokens;
 use regex::Regex;
+use vitasdk_sys_build_util::link_visitor::{
+    syn::{self, visit_mut::VisitMut},
+    Link,
+};
 
 fn main() {
     env_logger::init();
 
-    println!("cargo:rerun-if-env-changed=VITASDK");
-    let vitasdk = Utf8PathBuf::from(env::var("VITASDK").expect(
-        "Vitasdk isn't installed or VITASDK environment variable isn't set to a valid unicode",
-    ));
-    let sysroot = vitasdk.join("arm-vita-eabi");
-
-    assert!(
-        sysroot.exists(),
-        "VITASDK's sysroot does not exist, please install or update vitasdk first"
-    );
-
-    let lib = sysroot.join("lib");
-    assert!(lib.exists(), "VITASDK's `lib` directory does not exist");
-    println!("cargo:rustc-link-search=native={lib}");
```

Comment:
@ZetaNumbers removing `cargo:rustc-link-search=native={lib}` broke it for me and CI, as it's used to find the stubs when linking... Is it ok to revert commit https://github.com/vita-rust/vitasdk-sys/pull/11/commits/17653562ac19a53014d865c938906ffafd290ddb?